conformance/fhir-ig-davinci-pdex-plan-net/src/main/resources/hl7/fhir/us/davinci-pdex-plan-net/package/.index.json

```diff
@@ -6,21 +6,21 @@
       "resourceType": "ImplementationGuide",
       "id": "hl7.fhir.us.davinci-pdex-plan-net",
       "url": "http://hl7.org/fhir/us/davinci-pdex-plan-net/ImplementationGuide/hl7.fhir.us.davinci-pdex-plan-net",
-      "version": "0.3.0"
+      "version": "0.1.5"
```

Comment:
Should we add a release note warning?

Reply:
more specifically to the version change

Reply:
sure

Reply:
added the following in the draft release notes:
> Note that the previous packaged version of this IG was version `0.3.0`, but was subsequently "fixed" to indicate that this is still a revision of STU1 Ballot 1 and hence has been dropped to `0.1.5`